store/errors.go

```diff
@@ -49,6 +49,12 @@ var (
 
 	// ErrInvalidCredentials is returned on login error
 	ErrInvalidCredentials = errors.New("invalid credentials")
+
+	// ErrInvalidCredentials is returned when the user has not accepted the store's terms of service.
+	ErrTosNotAccepted = errors.New("terms of service not accepted")
+
+	// ErrNoPaymentMethod is returned when the user has no valid methods associated with their account.
+	ErrNoPaymentMethod = errors.New("no payment methods")
```

Comment:
Let's please also use plural `ErrNoPaymentMethods` on the variable name. It's a nitpick obviously, but it has a slightly different meaning in practice (there are none, rather than you didn't provide one).